stdlib/ballerina-http/src/main/ballerina/http/http_commons.bal

```diff
@@ -14,24 +14,46 @@
 // specific language governing permissions and limitations
 // under the License.
 
+documentation {
+    Options for HTTP chunking.
+
+    `AUTO`: If the payload is less than 8KB content-length header is set in the outbound response/response,
+            otherwise chunking header is set in the outbound response/response
```

Comment:
outbound response/response?

Reply:
also, shall we say "If the payload is less than 8KB, content-..."

Reply:
Yes. This is valid for both outbounds request & response.

Reply:
this says outbound **response/response**? :)

Reply:
So how would be the better way to say it?